<launch>
	<node type="image2ism.py" name="image2ism" pkg="image_inverse_sensor_model" respawn="true" output="screen">
		<!-- Select and object type (0-19 is YOLO darknet_ros, 20 is a pedestrian detector)  0.aeroplane, 1. bicycle, 2 bird, 3.boat, 4. bottle, 5. bus, 6. car, 7. cat, 8. chair, 9. cow, 10.diningtable 11. dog, 12. horse, 13. motorbike, 14. person, 15. pottedplant, 16. sheep, 17. sofa, 18. train, 19. tvmonitor, 20. pedestrian, continues -->
		<param name="objectTypeInt" value="32" />
		<param name="imageWidth" value="800" />
		<param name="imageHeight" value="600" />
		<param name="cam_xTranslation" value="0.2038" /> <!-- static transformations are made manually. Specifying the frame position according to the base of the frame -->
		<param name="cam_yTranslation" value="0.255" />
		<param name="cam_zTranslation" value="2.056" />
		<param name="cam_pitch" value="0.1963" />  <!-- 10.0 deg = 0.1745 rad, 11.25 deg = 0.1963 rad, 12.5 deg = 0.2182 rad -->
		<param name="cam_yaw" value="0.0" />
		<param name="cam_FOV" value="0.68067" />
		<param name="grid_resolution" value="0.5" />
		<param name="grid_xSizeInM" value="-1.0" />
		<param name="grid_ySizeInM" value="-1.0" />
	</node>
</launch>
